### PR TITLE
basic payments page working

### PR DESCRIPTION
**Pull request by ianopolous**

### CODE REVIEW

**Comment by ianopolous** [comment]
Screenshot_2020-03-23 Peergos(1)

File: src/components/payment/payment.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	      <br/>
+	      <div>
+		<ul style="text-align:left;padding-right:40px;">
+		  <li>10 GB of hyper secure storage</li>
+		  <li>5 GBP / month</li>
+		</ul>
+	      </div>
+            </button>
+	    <button class="btn btn-success" @click="requestStorage(107374182400)" style="height:10em;">
+              <div style="font-size:x-large">Visionary account</div>
```
Comment (kevodwyer) [inline src/components/payment/payment.html]: 3 options maybe?

Comment (ianopolous) [reply src/components/payment/payment.html]: Yep, the standard advice is to do 3 options, but what would our third be? We can't support 1 TiB level yet.

File: src/components/payment/payment.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <label v-if="isError">{{ error }}</label>
+	    <br/>
+	    <label> Current space: {{ getQuota() }}</label>
+	    <br/>
+            <button class="btn btn-success" @click="requestStorage(10737418240)" style="height:10em;">
+              <div style="font-size:x-large">Basic account</div>
+	      <br/>
+	      <div>
+		<ul style="text-align:left;padding-right:40px;">
+		  <li>10 GB of hyper secure storage</li>
```
[the remark this answers is not in the document]
Comment (ianopolous) [reply src/components/payment/payment.html]: Most people will need extra assurance.

**Pull request merged**